### ISSUE TITLE
AI moves siege unit next to a city

### ISSUE THREAD

**Issue by LukaSlovenia29** [LoneGazebo/Community-Patch-DLL#4017]
_1. Mod version (i.e Date - 4/23):_

2/27/2018

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 

The AI moved a siege unit next to my city. Specifically, Persia moved a catapult next to the city of Jericho on turn 124. Because the zip file is too big, I'll post the file on civfanatics.

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by ilteroi** [comment]
yeah, that happens sporadically. i've been working on some improvements, will post an experimental build once i'm back home

**Comment by LukaSlovenia29** [comment]
Thanks for the reply! I sent the logs&saves to Gazebo, I hope it will help with your work on improvements.

**Comment by ilteroi** [comment]
ok, here's the build. compatible with 2/27 but breaks savegames unfortunately.

try it and let me know if you notice any combat oddities.

[CvGameCore_Expansion2-release-20180304.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1779013/CvGameCore_Expansion2-release-20180304.zip)

**Comment by ilteroi** [comment]
ok, that made them a bit too timid. try this one instead.

[CvGameCore_Expansion2-release-20180305.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1782840/CvGameCore_Expansion2-release-20180305.zip)

**Comment by LukaSlovenia29** [comment]
Thanks! I'll try to get to it in the next few days, depending on work stuff.

**Comment by ilteroi** [comment]
now i got it.

[CvGameCore_Expansion2_release_20180306.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1786910/CvGameCore_Expansion2_release_20180306.zip)
